gitlab4j-api/src/main/java/org/gitlab4j/api/RepositoryApi.java

```diff
@@ -914,18 +914,7 @@ public File getRepositoryArchive(
      */
     public CompareResults compare(Object projectIdOrPath, String from, String to, boolean straight)
```

Comment:
I think this method should be deprecated, now that we have the one accepting more parameters.